x-pack/legacy/plugins/uptime/public/components/functional/donut_chart.tsx

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
why not move this into charts folder?

Reply:
yep, makes sense. I should've put the `WIP` label on this one 😄